code/ColladaExporter.cpp

```diff
@@ -1136,21 +1136,23 @@ void ColladaExporter::WriteJointsWeightSourceNode(const size_t pIndex){
 	delete [] tempBoneWeight;
 }
 
+template <typename T>
+static void moveMatrixInArray(const aiMatrix4x4t<T> &matrix, T *outBuffer){
+	std::memcpy(outBuffer,matrix[0],16*sizeof(T));
```

Comment:
No matrix[0] return the pointer to the data buffer for be able to use matrix[i][j].

see https://github.com/assimp/assimp/blob/master/include/assimp/matrix4x4.inl#L241